src/qt/bitcoin.cpp

```diff
@@ -460,6 +460,13 @@ int GuiMain(int argc, char* argv[])
 
     // Generate high-dpi pixmaps
     QApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
+
+#if defined(QT_QPA_PLATFORM_ANDROID)
+    QApplication::setAttribute(Qt::AA_DontUseNativeMenuBar);
+    QApplication::setAttribute(Qt::AA_DontCreateNativeWidgetSiblings);
+    QApplication::setAttribute(Qt::AA_DontUseNativeDialogs);
+#endif
```

Comment:
Why not??

Reply:
![image](https://user-images.githubusercontent.com/762502/97157083-eaa28a80-1777-11eb-8c08-d73e2291bd6b.png)
Native menus (or Qt's Android native implementation of QMenu) don't support submenus. When you tap on any of the four options nothing happens.

The other two attributes are there because native dialogs are way too simple to render our modals and to make sure the dialogs don't create another window (which is only supported on ChromeOS).